app/models/cart.rb

```diff
@@ -13,6 +13,8 @@ def add_item(item_id)
   def minus_item(item_id)
     if contents[item_id.to_s] > 0
       contents[item_id.to_s] -= 1
+    else contents[item_id.to_s] == 0
```

Comment:
Odd `else` layout detected. Did you mean to use `elsif`?<br>Operator `==` used in void context.